pkg/controller/kieapp/constants/constants.go

```diff
@@ -73,12 +73,18 @@ const (
 	LabelRHsubcomponentType = "rht.subcomp_t"
 	// LabelRHcompany used as metering label
 	LabelRHcompany = "com.company"
+	// IBMBamoeImageContext IBM image context
+	IBMBamoeImageContext = "ibm-bamoe"
+	// IBMBamoeImagePrefix IBM image prefix
+	IBMBamoeImagePrefix = "bamoe"
 	// RhpamPrefix RHPAM prefix
 	RhpamPrefix = "rhpam"
```

Comment:
Is still needed ?

Reply:
Yes, it is tied to a few things, image name, env prefix e a few other things.